packages/connectivity/src/scp-cf/destination/destination-service-cache.ts

```diff
@@ -1,27 +1,25 @@
 import { JwtPayload } from '../jsonwebtoken-type';
-import { Cache, IsolationStrategy } from '../cache';
+import { Cache } from '../cache';
 import { Destination } from './destination-service-types';
-import { getDestinationCacheKey } from './destination-cache';
+import { getDestinationCacheKey, IsolationStrategy } from './destination-cache';
 
 const DestinationServiceCache = (cache: Cache<Destination[]>) => ({
   retrieveDestinationsFromCache: (
     targetUrl: string,
-    decodedJwt: JwtPayload,
-    isolationStrategy?: IsolationStrategy
+    decodedJwt: JwtPayload
   ): Destination[] | undefined =>
     cache.get(
-      getDestinationCacheKeyService(targetUrl, decodedJwt, isolationStrategy)
+      getDestinationCacheKey(decodedJwt, targetUrl, IsolationStrategy.Tenant)
```

Comment:
I am also a bit surprised to see this. Wouldn't it be more "secure" if users could specify tenant_user here?

Also, if we leave it as is - doesn't that make the third parameter of `getDestinationCacheKey` obsolete?

Reply:
The cache is for the destinations only so the call where you get multiple destinations for the a given tenant. For this Tenant is the only meaningful isloation, because it does not depend on user. I wanted to use the destination cache so I passed the argument.

Reply:
Are you aware of https://github.com/SAP/cloud-sdk/issues/496 WRT user isolated caching? 

Or in general, is your cache upper-bounded already by the JWT lifetime (or any other authentication expiration time)?

Reply:
If the destination contains a token with some lifetime (expiresIn) the cache entry should go away the [token is expired](https://github.com/SAP/cloud-sdk-js/blob/8366f5d36d0de0edac0a15ab70b0ff02178d8635/packages/connectivity/src/scp-cf/destination/destination-cache.ts#L109) or after the default expiration time.

Reply:
ok great, in Java we currently don't have that (yet) unfortunately